src/test/data/JsonSerializableAddressBookTest/lengthyBooksAddressBook.json

```diff
@@ -0,0 +1,20 @@
+    "authors" : ["aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"]
+  }, {
+      "bookName" : "Lorem Ipsum comes from a latin text written in 45BC by Roman statesman Lorem Ipsum comes from a latin text written in 45BC by Roman statesman Lorem Ipsum comes from a latin text written in 45BC by Roman statesman Lorem Ipsum comes from a latin text written in 45BC by Roman statesman Lorem Ipsum comes from a latin text written in 45BC by Roman statesman Lorem Ipsum comes from a latin text written in 45BC by Roman statesman",
+      "isbn" : "0-545-01022-5",
+      "tagged" : [ "Detective", "Mystery", "Fantasy", "Classics", "Historical", "Horror", "Literary", "Romance", "Crime",
+        "SciFi", "Thrillers", "Drama", "Poetry", "Media", "Nonfiction", "Horror1", "Literary1", "Romance1", "Crime1",
+        "SciFi1", "Thrillers1", "Drama1", "Poetry1", "Media1" , "Media2", "Media3", "Media4", "Media5", "Media6",
+        "Media7", "Media8", "Media9", "Media10"],
+      "authors" : [ "accusamus acutum aegritudine alios aliquip allevatio antiqua arbitramur arbitrium arcu augeri captet censes cetero cogitavisse comparare conscientiam contentus cupiditatum deditum democritus depravata deserunt desiderant desistunt detracta didicerimus dirigentur displicet dissensio disseretur distinguique eget elit eo epicurei equos exercitus exorsus fastidium faucibus felis foedus fuissent generis geometrica gravida illam impetus incursione indignae inhaererent initia inquam inter ipsas ipsi iriure lacinia licet mel mi minim mors muniti netus noster nostro occultarum odioque omnino oratoribus oriantur ornateque percipit plura pluribus possint praetulerit privamur probamus procedat propemodum pulchraeque quaerendi quanti quisque rudem sensum sole stultorum successerit successionem suspicor tranquilli unde varias videre voce vulgo "]
```

Comment:
Nice edge case testing!

Reply:
Thank you, will update more edge cases to ensure that our program doesn't break.